Library/Formula/opentsdb.rb

```diff
@@ -0,0 +1,73 @@
+class Opentsdb < Formula
+  homepage "http://opentsdb.net"
+  url "https://github.com/OpenTSDB/opentsdb/releases/download/v2.1.0/opentsdb-2.1.0.tar.gz"
+  sha256 "b76e36a2d744170dde0ecd7ff3b937cbe68dd9fab9ce885aab5847de3e975157"
+
+  def install
+    ENV.deparallelize
+    system "./build.sh --prefix=#{prefix}"
+    system "mkdir -p #{prefix}/share/opentsdb/static/gwt/standard/images/ie6"
+    Dir.chdir("build") do
+      system "make", "install"
+    end
+  end
+
+  test do
+    system "true"
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!